misc/scripts/deploy-tools.sh

```diff
@@ -222,7 +222,7 @@ function cleanvm {
 function cleanupVMPre {
     case $FILE_EXT in
     deb)
-        $SSH $TARGET "$DEB_QUERY $PLUGIN_NAME > /dev/null"
+        $SSH $TARGET "$DEB_QUERY $PLUGIN_NAME > /dev/null 2>&1"
```

Comment:
this just hides error messages. Why are we getting them at the fist place?

Looking in the log ,this is likely another incarnation of CI race: 
```
tar: bin: time stamp 2017-06-25 17:52:50 is 936835.387593667 s in the future
cp config.json docker-volume-vsphere /tmp/plugin
cp: cannot stat 'config.json': No such file or directory
cp: cannot stat 'docker-volume-vsphere': No such file or directory
```
Meaning the messages are in  indication of a serious issues. 

I do not think we should turn them , we should fix the race